rules/windows/powershell/powershell_script/posh_ps_susp_local_group_reco.yml

```diff
@@ -1,4 +1,4 @@
-title: Suspicious Get Local Groups Information
+title: Suspicious Get Local Groups Information - Powershell Script Block Text
```

Comment:
`title: Suspicious Get Local Groups Information - PowerShell`
Like the other rules

Reply:
Modified